lib/link_thumbnailer/configuration.rb

```diff
@@ -65,6 +65,7 @@ def initialize
       @max_concurrency = 20
       @scrapers = [:opengraph, :default]
       @http_override_headers = { 'Accept-Encoding' => 'none' }
+      @download_size_limit = 10.megabytes
```

Comment:
you need to require `'active_support/core_ext/numeric/bytes'` for this to work

Reply:
Oh, good catch, thanks. Then I suggest to use `10 * 1024 * 1024` as default value instead, as people might not use it with Rails.